src/xcode/ENA/ENA/Source/Scenes/AntigenTestProfile/AntigenTestProfileInput/AntigenTestProfileInputViewController.swift

```diff
@@ -144,9 +144,18 @@ class AntigenTestProfileInputViewController: UITableViewController, FooterViewHa
 	
 	func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
 		// filtering out emojis and any other unwanted charaters. these are not wanted in the test profile.
+		var string = string
+		
+		if string.last?.isWhitespace == true { // possible keyboard suggestion
```

Comment:
```suggestion
		if string.last?.isWhitespace { // possible keyboard suggestion
```

Reply:
That cannot work. It will result in a compile error.